main.lua

```diff
@@ -36,6 +36,23 @@ local TextEventType = {
     REPLACE = 0
 }
 
+local LSPErrorCodes = {
+	[-32700] = "ParseError",
+	[-32600] = "InvalidRequest",
+	[-32601] = "MethodNotFound",
+	[-32602] = "InvalidParams",
+	[-32603] = "InternalError",
+
+	[-32099] = "serverErrorStart",
+	[-32002] = "ServerNotInitialized",
+	[-32001] = "UnknownErrorCode",
+	[-32000] = "serverErrorEnd",
```

Comment:
`serverErrorStart`/`serverErrorEnd` are not names for error codes, they indicate a reserved range of error codes.
```suggestion
	[-32002] = "ServerNotInitialized",
	[-32001] = "UnknownErrorCode",
```